doc/source/whatsnew/v0.25.0.rst

```diff
@@ -35,6 +35,7 @@ Other Enhancements
 - :class:`RangeIndex` has gained :attr:`~RangeIndex.start`, :attr:`~RangeIndex.stop`, and :attr:`~RangeIndex.step` attributes (:issue:`25710`)
 - :class:`datetime.timezone` objects are now supported as arguments to timezone methods and constructors (:issue:`25065`)
 - :meth:`DataFrame.query` and :meth:`DataFrame.eval` now supports quoting column names with backticks to refer to names with spaces (:issue:`6508`)
+- :class:`CustomBusinessMonthBegin` and :class:`CustomBusinessMonthEnd` now use the substitution pattern for docstring sharing
```

Comment:
This is an internal refactor and not visible to end users so no whatsnew required

Reply:
Removed !